test/icmpv6/src/nf.rs

```diff
@@ -0,0 +1,90 @@
+use colored::*;
+use netbricks::headers::*;
+use netbricks::operators::*;
+use std::default::Default;
+use std::net::Ipv6Addr;
+use std::str::FromStr;
+
+struct Meta {
+    msg_type: IcmpMessageType,
+    code: u8,
+    checksum: u16,
+    src: Ipv6Addr,
+    dst: Ipv6Addr
+}
+
+impl Default for Meta {
+    fn default() -> Meta {
+        Meta {
+            msg_type: IcmpMessageType::NeighborAdvertisement,
+            code: 0,
+            checksum: 0,
+            src: Ipv6Addr::unspecified(),
```

Comment:
do we need these as meta? I know it's kinda a banal exercise, but maybe just having what's in the ipv6 header already works.

Reply:
i cleaned up the test to only check for the icmpv6 fields that we wanted to compare. i removed the src and dist address in the ipv6 header in the meta struct